File: app/src/main/java/com/example/msdc/api/TVResponse.java
package com.example.msdc.api;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class TVResponse {
    @SerializedName("results")
    private final List<TVResult> results = new ArrayList<>();

    public List<TVResult> getResult(){ return results; }
}

File: app/src/main/java/com/example/msdc/api/TVResponse.kt
package com.example.msdc.api

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class TVResponse (
    @SerializedName("results") val result: ArrayList<TVResult>? = null
) : Parcelable